### PR TITLE
[keymap] Display logo for caps lock indication on crkbd using old OLED driver

### PR DESCRIPTION
**Pull request by manna-harbour**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/manna-harbour_miryoku/rules.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
-
-# generated from users/manna-harbour_miryoku/miryoku.org
+# generated from users/manna-harbour_miryoku/miryoku.org  -*- buffer-read-only: t -*-
 
 MOUSEKEY_ENABLE = yes        # Mouse keys(+4700)
 EXTRAKEY_ENABLE = yes        # Audio control and System control(+450)
+EXTRAFLAGS += -flto	     # Link Time Optimization to reduce code size, 31358->28034/28672
```
Comment (drashna) [inline users/manna-harbour_miryoku/rules.mk]:
```suggestion
LTO_ENABLE = yes
```

This enables the same option, and disables action_macros, and action_functions, both of which seem to break when LTO is enabled.   And this is a lot simpler to use.

**Comment by manna-harbour** [comment]
> However, I would rather see the OLED Driver implementation over the SSD1306 implementation.

Thanks, I would prefer it too except it doesn't seem fully compatible with the crkbd yet. In particular timeout doesn't work and I couldn't adapt your work around, clear doesn't work, and the display stays on after the host suspends.  I will change over after support improves or I have time to work on the work-arounds.

**Comment by drashna** [comment]
Sounds good

**Comment by manna-harbour** [comment]
I think I have the new driver working satisfactorily now.  I can push it to this PR but if the commit messages are squashed as is it'll be a bit of a mess.  Can I force-push to this PR or should I create a new one?

**Comment by manna-harbour** [comment]
Thanks drashna!

**Pull request merged**